packages/tabs/src/Tabs.stories.tsx

```diff
@@ -110,6 +110,26 @@ export const LiveExample: StoryFn<TabsProps> = ({
   </LeafyGreenProvider>
 );
 
+export const Controlled: StoryFn<TabsProps> = (args: TabsProps) => {
+  const [selectedTab, setSelectedTab] = useState(0);
+
+  return (
+    <LiveExample
+      {...args}
+      selected={selectedTab}
+      setSelected={setSelectedTab}
```

Comment:
Since we're controlling this value, can we remove it from the SB "controls" 
![Screenshot 2024-05-30 at 1 41 00 PM](https://github.com/mongodb/leafygreen-ui/assets/2414030/904c1c25-4e70-4ed2-affc-60e95456ac1a)